tests/e2e/chain_init/main.go

```diff
@@ -5,19 +5,38 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"time"
 
 	"github.com/osmosis-labs/osmosis/v7/tests/e2e/chain"
 )
 
 func main() {
 	var (
-		dataDir string
-		chainId string
+		valConfig []*chain.ValidatorConfig
+		dataDir   string
+		chainId   string
 	)
 
 	flag.StringVar(&dataDir, "data-dir", "", "chain data directory")
 	flag.StringVar(&chainId, "chain-id", "", "chain ID")
+
 	flag.Parse()
+	file := fmt.Sprintf("%v/%v-configEncode", dataDir, chainId)
+	for i := 0; i < 10; i++ {
```

Comment:
Why is this needed? Is it because we are mounting on a volume? 

If yes, can we instead pass a string via CLI? These files are expected to be quite small. I think it would be a cleaner solution than doing repeats